### DISCUSSION TITLE
Feature Request: Add ansi color functions

### DISCUSSION THREAD

**Discussion by stelcodes** [babashka/cli#45] [Ideas]
Hi Micheal!
I'm wondering if you'd be interesting in adding a namespace for ansi color functions like `io.aviso.pretty.ansi`? I love adding color to my cli scripts but I don't think any ansi helpers are included with Babashka at the moment, but I could be wrong. Also copying `clansi` is another option or maybe including it as a default library with Babashka? Idk what do you think?

Reference:
https://github.com/AvisoNovate/pretty/blob/master/src/io/aviso/ansi.clj
https://github.com/ams-clj/clansi/blob/master/src/clansi/core.clj

**Comment by borkdude** [comment]
Hey @stelcodes - I don't know if this belongs in babashka.cli but you are aware that you can use these libraries as `:deps` in `bb.edn` right?
